src/http/nodegen/middleware/applicationMiddleware.ts

```diff
@@ -44,18 +43,42 @@ export const requestParser = (app: express.Application): void => {
 
   // parse the body
   app.use(express.urlencoded({ extended: false }));
-
-  app.use(requestIp.mw());
```

Comment:
Now the IP is completely removed from the req object - this is breaking change for any IP grabbing the IP from the req.

Reply:
Ah I missed that, I thought the only thing it did was add it to logs. 

Can just add it back in.